test/screenshot/dialog/alert.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import '../../../packages/top-app-bar/index.scss';
+import Dialog, {
+  DialogContent,
+  DialogFooter,
+  DialogButton,
+} from '../../../packages/dialog/index';
+import Button from '../../../packages/button/index';
+import './index.scss';
+
+
+class Alert extends React.Component<{}, {isOpen: boolean; action: string;}> {
+  state = {isOpen: true, action: ''};
+
+  render() {
+    return (
+      <main className='mdc-layout-grid'>
+        <aside>
+          <Button
+            raised
+            onClick={() => this.setState({isOpen: !this.state.isOpen})}
+          >{this.state.isOpen ? 'close dialog' : 'open dialog' }</Button>
+          <p className='mdc-typography--body1'>Dialog Action:
```

Comment:
If you have time - change this to use the Typography component (and in other places). Not a big deal since its tests.